images/inference-services/kfserving/run.sh

```diff
@@ -37,25 +40,63 @@ case $PREDICTOR in
     # named '1', create it and copy the tensorflow model to it.
     tensorflow)
         if [ "$(mc find ${model_bucket} --name 1)" = "" ]; then
-            mc cp -r ${model_bucket}/tfmodel/ ${model_bucket}/1
+            mc cp -r "${model_bucket}"/tfmodel/ "${model_bucket}"/1
         fi
-        export RUNTIME_VERSION=$(mc cat ${model_bucket}/requirements.txt | awk -F '=' '/tensorflow/ {print $3}')
+        export RUNTIME_VERSION=$(mc cat "${model_bucket}"/requirements.txt | awk -F '=' '/tensorflow/ {print $3}')
         prediction_url_path="${isvc}:predict"
         ;;
     # kfserving expects the sklearn model file as model.joblib however mlflow
     # saves the model as model.pkl, so if there is no model.joblib, create a
     # copy named model.joblib from model.pkl
     sklearn)
-        if ! mc ls ${model_bucket} | grep -q "model.joblib"; then
-            mc cp ${model_bucket}/model.pkl ${model_bucket}/model.joblib
+        if ! mc ls "${model_bucket}" | grep -q "model.joblib"; then
```

Comment:
Just out of curiosity, why were these quotes needed ?

Reply:
I think this will do better than me explaining 😄  https://github.com/koalaman/shellcheck/wiki/SC2086

If you are using VSC you can install the `shellcheck` extension, I did it recently and got a lot of warnings from this script so this is just an attempt to fix some of the easy ones.

We could also consider running it as part of the CI.